### DISCUSSION TITLE
Attributes is empty

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11817] [Q&A]
apm: 9.6
java-agent: 9.0
asianux server  7.3

I deleted all indexes and templates and restarted the oap service; Attributes has no data;

<img width="819" alt="image" src="https://github.com/apache/skywalking/assets/29425860/73f9961a-d285-464f-ab47-a9d91e377d41">

**Comment by wu-sheng** [comment]
The properties are only reported in a several-minute period. If you don't reboot Java service side, you need to wait.

**Reply by mitht** [reply]
I have restarted the Java server;
The overview trace has data; After waiting for a long time, there is still no data.

Check the es index data; there is no data in the properties field;
The system I am using is asianux 7.3. Is it because the system is incompatible?


index: sw_metrics-all
{
  ...
  "metric_table":"instance_traffic",
  "properties": ""
}

**Reply by wu-sheng** [reply]
I am not aware of any issue to cause this. I am new to asianux Linux system, but this is a Java env, it should be irrelevant.

**Reply by mitht** [reply]
-Dskywalking.agent.instance_properties_json={"systemId":"branch-db-01-011","appId":"app-00096"}

What has happened to this parameter? If this parameter is added, this problem will occur.
agent 9.0

**Reply by wu-sheng** [reply]
I don't use this, but the agent should be able to report libs and other information automatically.
